guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -66,3 +66,9 @@ For BIOS provisioning, you must associate a PXELinux template with the operating
 For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
 If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
 endif::[]
+
+.Bonded network interfaces
+You can configure a bonded interface on your host and use it during PXE based provisioning after the actual PXE boot.
```

Comment:
Maybe to specify it a bit more explicitly:
`You can configure a bonded interface to be used during the installation process (for downloading installation sources e.t.c.) and for the provisioned system (to be used after the provisioning process is finished).`

Reply:
```suggestion
You can configure a bonded interface to be used during the installation process, such as for downloading installation content, and for the provisioned system to be used after the provisioning process is finished.
```